src/components/Articles.jsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+import { fetchArticles } from "../api";
+
+const ArticleList = () => {
+  const [articles, setArticles] = useState([]);
+//   const [isLoading, setIsLoading] = true;
+
+  useEffect(() => {
+    fetchArticles().then((articleData) => {
+      setArticles(articleData);
+      console.log(articleData)
+      console.log(articles)
+      
+    //   setIsLoading(false);
+    });
+  }, []);
+
+//   if (isLoading) {
+//     return <div>Loading...</div>;
```

Comment:
you've got the code for the "loading..." screen commented out, but you do need this in the app! 
Sometimes it takes a bit of playing around with to get it working, check over the notes and previous repos if you need to see it again! 👍